code/src/index.css

```diff
@@ -1,13 +1,88 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  background-color: rgb(255, 144, 162);
+  font-family: 'Roboto', sans-serif;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.card {
```

Comment:
Someone had added a suuuuperlong word as a message just as I was looking at your code, and then I realized that it spilled over on your card (It did on my page too, so I have to fix that as well 😝) 
Apparently, you can add 
word-wrap: break-word;
to the card to make it break up the words that don't fit!

Reply:
Such a great tip, thanks Amanda!! Updated now when revisiting the project :-)